.htaccess

```diff
@@ -69,6 +69,8 @@ deny from env=stayout
 
 	# redirect all trafic to https
 	# Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS
+	# Header always edit Set-Cookie (.*) "$1; HTTPOnly; Secure"
+	# Header set Strict-Transport-Security: "max-age=63072000"
```

Comment:
Doesn't this conflict with `Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS`at line 71?